### PR TITLE
Client base setup

### PR DESCRIPTION
**Pull request by OmiCoding**
Made some additional changes
 - Added animation to launch.
 - Created new file for starry field animation.
 - Made some changes to the theme file.

### CODE REVIEW

File: views/launch.ejs [hunk 1]
```diff
@@ -21,6 +21,9 @@
     <link href="css/header.css" rel="stylesheet" type="text/css" />
     <link href="css/footer.css" rel="stylesheet" type="text/css" />
     <link href="css/launch.css" rel="stylesheet" type="text/css" />
+    <<<<<<< HEAD
+    <link href="css/stars.css" rel="stylesheet" type="text/css" />
+    ======= >>>>>>> main
```
Comment (HermanDhillon) [inline views/launch.ejs]:
```suggestion
remove head and main merge conflicts
```

**Review by HermanDhillon — CHANGES_REQUESTED** [review]
make simple deletion

File: views/launch.ejs (hunk 1 above)
Comment (OmiCoding) [reply views/launch.ejs]: Understood, no problem!

**Review by HermanDhillon — APPROVED** [review]
looks good

**Pull request merged**